ee/modules/600-flant-integration/docs/README_RU.md

```diff
@@ -103,3 +93,99 @@ kubectl annotate node test pricing.flant.com/nodeType=Special
+
+#### Оповещения в Madison
+
+Madison — сервис обработки оповещений в составе платформы мониторинга компании Флант. Madison может принимать уведомления в формате Prometheus. 
+
+При создании нового кластера Deckhouse:
+1. При помощи ключа лицензии происходит автоматическая регистрация кластера в Madison.
+2. Madison выдаёт кластеру ключ, необходимый для отправки уведомлений.
+3. При помощи DNS-запроса для домена `madison-direct.flant.com` Deckhouse находит все доступные на текущий момент IP-адреса Madison.
+4. Для каждого адреса создается Pod `madison-proxy`, в который Prometheus отправляет уведомления.
+
+Схема отправки оповещений из кластера в Madison:
+![](../../images/600-flant-integration/image5.png)
+
+В среднем скорость отправки оповещений из кластера равняется **2kb/s**. Но здесь следует учитывать, что чем больше инцидентов произошло в кластере, тем больше данных будет отправлено.
+
+#### Логи оператора Deckhouse
+Оператор Deckhouse является центральным компонентом всего кластера. Для того чтобы иметь данные для отладки во всех обслуживаемых кластерах, в операторе Deckhouse организована отправка логов при помощи модуля [log-shipper](../460-log-shipper/) в хранилище Loki компании Флант (не напрямую, а также через сервис Connect).
+
+В логах **нет секретных данных** касающихся кластера (примеры сообщений приведены на рисунке ниже). Собранная информация помогает понять, какие действия выполнял оператор и когда он их выполнял.
+
+Пример логов оператора Deckhouse:
+![](../../images/600-flant-integration/image4.png)
+
+При смене релиза Deckhouse, количество отправляемых данных логов достигает в среднем **150 записей в минуту**. При обычной работе, количество отправляемых данных логов достигает в среднем **20 записей в минуту**.
+
+#### Метрики SLA
+
+С помощью модуля [upmeter](../500-upmeter/) в кластере выполняется измерение уровня доступности различных компонентов. Такие метрики необходимы для определения уровня доступности компонентов платформы и анализа выполнения соглашения об уровне сервиса (SLA, Service Level Agreement).  
+
+Одна из особенностей модуля upmeter заключается в том, что он может отправлять метрики SLA по протоколу [Prometheus Remote Write](https://docs.sysdig.com/en/docs/installation/prometheus-remote-write/).
+
+Интерфейс upmeter:
+![](../../images/600-flant-integration/image2.png)
+
+Графики по метрикам из upmeter в Grafana:
+![](../../images/600-flant-integration/image3.png)
+
+В среднем модуль upmeter отправляет **70 строк каждые 30 секунд**.
+
+### Как отключить отправку данных Deckhouse? 
+
+Чтобы отключить отправку данных, необходимо:
+- Отключить модуль `flantIntergation`, чтобы отключить регистрацию в Madison и отправку метрик.
+- Удалить CustomResource [UpmeterRemoteWrite](../500-upmeter/cr.html#upmeterremotewrite), чтобы отключить отправку [метрик доступности](#метрики-sla) модулем [upmeter](../500-upmeter/).
```

Comment:
@nabokihms Am I right here?

Reply:
It's enough disable `flant-integration` module. 

When user disables `flant-integration` module, its object are to be deleted by deckhouse including `UpmeterRemoteWrite`. With that, Upmeter stops exporting metrics.  After deckhouse converges all modules, there will be no module-related `UpmeterRemoteWrite` object to delete.

Reply:
Thanks. Refactored.